### PR TITLE
Completed html-syntax

### PR DESCRIPTION
**Pull request by theDanielYeh**
image

### CODE REVIEW

**Comment by theDanielYeh** [comment]
Oops, I realized I put "Pikachu" as the name...

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+
+<html>
+
+  <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/890.png" alt="A pokemon." title="A mosntrous pokemon.">
```
Comment (BrettAlbright) [inline html-syntax/index.html]: make sure that all tags which you want to display on the screen are within the body tags

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed on img tag location

File: html-syntax/index.html (hunk 1 above)
Comment (theDanielYeh) [reply html-syntax/index.html]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**